src/packages/@workspaces/features/rest-explorer/layout/RestExplorer.svelte

```diff
@@ -127,7 +128,12 @@
 </script>
 
 {#if $tab.tabId}
-  <div class="d-flex rest-explorer-layout">
+  <div
+    class="d-flex rest-explorer-layout"
+    style="height: {isLoginBannerActive
+      ? 'calc(100vh - 126px)'
+      : 'calc(100vh - 80px)'};"
+  >
     <div class="w-100 d-flex flex-column h-100 p-3">
       <!-- Request Name Header -->
```

Comment:
we have a seperate layout file for rest explorer where we maintain the layout of the rest explorer, this should be applied on that file only